Styles/styles.css

```diff
@@ -0,0 +1,233 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto+Mono');
+
+/************
+*Site Header*
+*************/
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+  background: white;
+}
+
+header {
+  position: fixed;
+  background: #fff;
```

Comment:
Clever! I didn't learn about abbreviating hex colors that had the same letter until much later.